src/worker/queues/logQueue.ts

```diff
@@ -0,0 +1,14 @@
+import { Queue } from "bullmq";
+import { redis } from "../../utils/redis/redis";
+import { defaultJobOptions } from "./queues";
+
+export class LogQueue {
```

Comment:
Can we use a clearer name? I have no idea what this does based on the name? Ideally something like `<verb><noun>Queue`

Reply:
renamed it to `nonceHealthCheckQueue`, open to suggestions